src/DotNet.Glob/Evaluation/AnyCharacterTokenEvaluator.cs

```diff
@@ -11,6 +11,7 @@ public AnyCharacterTokenEvaluator(AnyCharacterToken token)
         {
             _token = token;
         }
+
```

Comment:
Can we revert this line space change, as it's unrelated. I am most likely holding you to a higher standard of PR than I hold myself..